backend/src/services/v2/smtp/smtp.ts

```diff
@@ -86,6 +88,88 @@ export async function requestReviewForAccessRequest(
   await Promise.all(sendEmailResponses)
 }
 
+export async function requestResponsePostReleaseReviewResponse(review: ReviewDoc, release: ReleaseDoc) {
```

Comment:
Unlike the other previous emails we've added here, these emails aren't _requesting_ the recipient to do something. Maybe `notifyReviewResponseForRelease` instead, although that is along a bit long.